packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -343,6 +345,7 @@ class _HelperErrorState extends State<_HelperError> with SingleTickerProviderSta
           style: widget.errorStyle,
           textAlign: widget.textAlign,
           overflow: TextOverflow.ellipsis,
```

Comment:
Do you also want to be able to specify the overflow here?  I know that at least for Arabic, ellipsis is pretty bad as a choice (it cuts off whole words only, so often doesn't fill the available space).  It's a good default, but it would be good to be able to specify it. (See https://github.com/flutter/flutter/issues/16450 also)

Reply:
That seems reasonable. An alternative would be an error widget value, which would would allow one to display anything at all for the error. The implementation is already structured this way. The hardest part might be deciding what to call the property. Maybe `InputDecoration.errorWidget`.

If it's OK with you, I'd like to defer this bit of evolution to a separate PR.

Reply:
Sure.